src/ralph/accounts/views.py

```diff
@@ -28,6 +31,51 @@ def user_licence(self, item):
             return []
 
 
+class InventoryTagConfirmationView(RalphBaseTemplateView):
+    template_name = 'ralphuser/inventory_tag_confirmation.html'
+
+    def get_context_data(self, **kwargs):
+        print(self.request.method)
+        context = super().get_context_data(**kwargs)
+        asset_fields = [
+            ('barcode', _('Barcode / Inventory Number')),
+            'model__category__name', 'model__manufacturer__name',
+            'model__name', ('sn', _('Serial Number'))
+        ]
+
+        context['asset_details'] = MyEquipmentAssetList(
+            BackOfficeAsset.objects.filter(id=self.kwargs['asset_id']),
+            asset_fields,
+            request=self.request
+        )
+        return context
+
+
+class InventoryTagView(RalphBaseTemplateView):
+    template_name = 'ralphuser/inventory_tag.html'
+    
+    def post(self, request, *args, **kwargs):
+        if not 'asset_id' in request.POST:
+            return HttpResponseBadRequest()
```

Comment:
IMO this block is redundant - because [this line](https://github.com/allegro/ralph/pull/2602/files#diff-8684faeb9511ab3569426359d236c9f7R18) care of this